### PR TITLE
Create version for Silex 2.0

### PR DESCRIPTION
**Pull request by SpacePossum**
Preparing for Silex 2 [coming up](https://github.com/silexphp/Silex/issues/1314#issuecomment-217094831).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
   "license": "MIT",
   "keywords" : ["Silex", "Config", "Configuration", "Yaml"],
   "require": {
-    "php": "^5.4 || ^7.0",
-    "silex/silex": "~1.1",
+    "php": "^5.5 || ^7.0",
+    "silex/silex": "dev-master",
```
Comment (SpacePossum) [inline composer.json]: wait for the 2.x release

File: composer.json [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
     "symfony/filesystem": "2.7.3"
   },
   "require-dev": {
     "symfony/yaml": "~2.6",
-    "gecko-packages/gecko-memcache-mock" : "1.*"
+    "gecko-packages/gecko-memcache-mock" : "^2.0"
```
Comment (SpacePossum) [inline composer.json]: should be reverted, there is no need.

**Comment by SpacePossum** [comment]
All green, @tiemevanveen do you want to play around with this otherwise I'll continue to merge this and prep. a v2 release :)

**Comment by tiemevanveen** [comment]
Sure, don't wait on me, just merge, I'll check it later!

**Comment by tiemevanveen** [comment]
\0/

**Comment by SpacePossum** [comment]
I'll branch and tag later :)

I'm thinking about bumping the SF packages req. to 3.x on 2.x here. While there is no real need now (I think) I don't want to support two major versions of those on two major branches here... not sure yet...

**Comment by tiemevanveen** [comment]
Sounds like a good idea. Bump often and early :D

**Pull request merged**